testsuites/CBLTester/topology_specific_tests/multiple_sync_gateways/test_sgreplicate_cbl.py

```diff
@@ -1950,7 +1950,7 @@ def test_sg_replicate_restart_active_passive_nodes(params_from_base_test_setup,
                 if doc["updates-cbl"] != sg1_doc["updates-cbl"]:
                     replication_successful_flag = False
                     break
-            except KeyError as e:
+            except KeyError:
```

Comment:
Did you run flake8? I think this is causing an error in recent flake8 (but maybe I'm wrong there)

Reply:
Yes, it was as 'e' was unused.  The new versions of flake8 take this into considering. It is worth removing anything that is not used at all.